symplyphysics/laws/thermodynamics/relative_humidity.py

```diff
@@ -1,32 +1,31 @@
-from sympy import (Eq, solve, S)
+from sympy import (Eq, solve)
 from symplyphysics import (
     units,
     Quantity,
     Symbol,
     print_expression,
     validate_input,
     validate_output,
-    dimensionless,
-    convert_to
+    dimensionless
 )
 
 # Description
 ## Relative humidity is the ratio of water vapor pressure p to saturated vapor pressure p0 at a given temperature.
+## Saturated vapor is vapor in dynamic equilibrium with a liquid or solid of the same composition in a closed system.
+## In such a pair, the number of evaporating molecules is equal to the number of condensing molecules per unit of time.
+## Water vapor is the gaseous aggregate state of water.
```

Comment:
Of course. But the important part here is that it is this vapor, that we measure humidity of.